src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -44,6 +44,7 @@ public class DirectTransferGenerator implements GraphBuilderModule {
   private final Duration radiusByDuration;
 
   private final List<RouteRequest> transferRequests;
+  private final boolean noTransfersOnIsolatedStops;
```

Comment:
There is a miss match in the domain language here. In station we added the feature for turning transfers off by the `Station#transfersNotAllowed` - but here we are interpreting that as an isolated stop. If we want to do it this way the flag int the Station must be named `isolated`.

But do the `DirectTransferGenerator` need to know about this feature? Is it enough if the import check the config setting(`noTransfersOnIsolatedStops`) and only set the Station#transfersNotAllowed if the setting is `true`. If this is not the case, then we need to change the config, because the config apply to transfer generation, not import.

Reply:
We decided in the dev meeting, that `transfersNotAllowed` should only be set, if the feature is enabled in the build config